code/modules/admin/secrets.dm

```diff
@@ -546,6 +547,23 @@
 				B.update_hair()
 			message_admins("[key_name_admin(usr)] activated dorf mode")
 
+		if("schoolgirl")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","SG")
+			for(var/obj/item/clothing/under/W in world)
```

Comment:
SInce it's not going to be pressed that often, the lag shouldnt be that bad (hopefully?). I don't know how to make a global list of all clothing though.